pkgs/applications/networking/droopy/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, buildPythonApplication }:
+
+with lib;
+
+buildPythonApplication rec {
+  name = "droopy-${version}";
+  version = "20160830";
+
+  src = fetchFromGitHub {
+    owner = "stackp";
+    repo = "Droopy";
+    rev = "7a9c7bc46c4ff8b743755be86a9b29bd1a8ba1d9";
+    sha256 = "03i1arwyj9qpfyyvccl21lbpz3rnnp1hsadvc0b23nh1z2ng9sff";
+  };
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+  installPhase = ''
```

Comment:
What about just:

```sh
install -vD man/droopy.1 "$out/share/man/man1/droopy.1"
install -vD droopy "$out/bin/droopy"
```

It's much more concise and also doesn't copy files to the wrong directories (like the wrong dir for the manpage) and cause artifacts (like "$out/droopy").